lib/presentation/view/modal_sheets/confirm_modal_sheet_builder.dart

```diff
@@ -75,6 +75,7 @@ class ConfirmModalSheetBuilder {
 
   void show(context) {
     showModalBottomSheet(
+        useRootNavigator: true,
```

Comment:
Can you explain it?

Reply:
![image](https://user-images.githubusercontent.com/48354990/105328194-1515af80-5bd0-11eb-873e-40617aee95e6.png)

It's to put the context menu widget in first layer